client.go

```diff
@@ -120,7 +120,7 @@ func verifyFile(c *cli.Context) error {
 	if len(c.Args().First()) == 0 {
 		printErrAndExit("Please give the 'msgFile'", 1)
 	}
-	dbg.SetDebugVisible(c.GlobalInt("debug"))
+	log.SetDebugVisible(c.GlobalInt("debug"))
```

Comment:
This doesn't really have to do with `ui` and  `dbg`; it's about logging and debug output. Also I was always in favour of not creating our own logging framework in the first place.